src/commands/dev-env-sync-sql.ts

```diff
@@ -23,7 +25,7 @@ import { getReadInterface } from '../lib/validations/line-by-line';
  * @return Site home url. null if not found
  */
 function findSiteHomeUrl( sql: string ): string | null {
-	const regex = "'(siteurl|home)',\\s?'(.*?)'";
+	const regex = `['"](siteurl|home)['"],\\s?['"](.*?)['"]`;
```

Comment:
Minor regex change because mydumper uses double quotes while mysqldump uses single quote.

Using two regexes may be more accurate, but this change in the regex seems to be good enough from my testing.